src/main/java/com/example/springcrud/entity/CommentEntity.java

```diff
@@ -0,0 +1,24 @@
+package com.example.springcrud.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+import lombok.*;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Table(name = "comment")
+public class CommentEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+    private String content;
+    @JsonIgnore
```

Comment:
Daca in Entity apare necisitatea de a face modificare in Json, e mai bine de creat un alt Dto deja. Mai mult ca atat, tu deja ai unul ce de fapt si este acest entity faca Product, e CommentDto.

In general se considera, ca in logica trebuie sa fie lucru practic doar cu Dto, folosind Entity doar pentru CRUD (create, read, update, delete). Deoarece baza de date se poate schimba, pot aparea coloane noi, dar logica oricum trebuie sa lucreze la fel.